### PR TITLE
[php] ask openssl to ignore no EOL

### PR DESCRIPTION
**Pull request by Lex-2008**

### CODE REVIEW

**Comment by cf-bottom** [comment]
Thank you for submitting a pull request! Maybe @craigcomstock can review this?

File: deps-packaging/php/cfbuild-php.spec [hunk 1]
```diff
@@ -6,6 +6,7 @@ Version: %{version}
 Release: 1
 Source0: php-%{php_version}.tar.gz
 Source1: php.ini
+Patch0: openssl-eol.patch
```
Comment (craigcomstock) [inline deps-packaging/php/cfbuild-php.spec:9]: why is it called "eol" patch and not "eof" or "openssl-ignore-unexpected-eof.patch" even to be horribly verbose?

**Comment by Lex-2008** [comment]
indeed! Actually, that's a wrong PR and should be closed. Thanks for noticing and sorry for the noise!

@cf-bottom bad bot

**Comment by cf-bottom** [comment]
I'm not sure I understand, @Lex-2008.

**Comment by craigcomstock** [comment]
@cf-bottom I think what @Lex-2008 is trying to say to you is that this PR is invalid and should be closed. Right @Lex-2008 ? ;)

**Comment by cf-bottom** [comment]
I'm not sure I understand, @craigcomstock.

**Pull request closed without merging**